pkg/common/config/config.go

```diff
@@ -47,6 +47,10 @@ const (
 	DefaultGCConfigPath = "/etc/cloud/pvcsi-config/cns-csi.conf"
 	// EnvVSphereCSIConfig contains the path to the CSI vSphere Config
 	EnvVSphereCSIConfig = "VSPHERE_CSI_CONFIG"
+	// EnvFeatureStates contains the path to the CSI Feature States Config
+	EnvFeatureStates = "FEATURE_STATES"
+	// DefaultVanillaFeatureStateConfigPath is the default path of csi feature states config file in Vanilla Cluster
+	DefaultVanillaFeatureStateConfigPath = "/etc/cloud/csi-feature-states/csi-feature-states.conf"
```

Comment:
IMO, from the next release onwards, this feature should be enabled. Why should we expect the user to add another `conf` file just to enable migration. The process is meant to be seamless.

Reply:
yes I agree

Reply:
I put second thought on this. Just in case if a user is not creating feature state configmap using filename - `csi-feature-states.conf` this will be useful to specify the correct path of the feature state file.

```
kubectl create configmap csi-feature-states --from-file=csi-feature-states.conf  --namespace=kube-system
```